boot/zephyr/main.c

```diff
@@ -500,7 +491,7 @@ void main(void)
                    CONFIG_BOOT_USB_DFU_DETECT_PIN_VAL,
                    CONFIG_BOOT_USB_DFU_DETECT_DELAY)) {
 #ifdef CONFIG_MCUBOOT_INDICATION_LED
-        gpio_pin_set(led, LED0_GPIO_PIN, 1);
+        gpio_pin_set_dt(led0, 1);
```

Comment:
```suggestion
        gpio_pin_set_dt(&led0, 1);
```
I am not familiar with this API, but it seems that `gpio_pin_set_dt` expects a pointer to a `struct gpio_dt_spec` instance.